index.js

```diff
@@ -31,6 +35,21 @@ function plugin (fn, options) {
   return fn
 }
 
+function checkName (fn) {
+  if (fn.name.length > 0) return fn.name
+
+  const r1 = new RegExp('at\\s{1}plugin\\s{1}.*\\n\\s*(.*)')
+  const r2 = new RegExp('\\/(\\w*)\\.js')
```

Comment:
Can you move those two regexps at the top of the file and define them with more clear names?